### DISCUSSION TITLE
Why breadcrumb warping on low space?

### DISCUSSION THREAD

**Discussion by sasgosh** [quasarframework/quasar#9388] [General - Components / Directives / etc] [answered]
Hello,

I have breadcrumb and buttons in row (left and right). When there is low space, I want to be up and down. Everything I tried didn’t work properly.

```
<q-card bordered flat class="row q-ma-sm q-pa-sm background">
    <q-item-section class="col-auto-width">
        <q-breadcrumbs class="no-warp">
            <q-breadcrumbs-el icon="home" to="/" />
            <q-breadcrumbs-el label="Something" />
        </q-breadcrumbs> 
    </q-item-section>
    <q-item-section class="col-auto">
        <div class="no-warp">
            <q-btn round outline dense color="primary" icon="sync" size="sm" class="q-mr-xs"></q-btn>
            <q-btn round outline dense color="primary" icon="add" size="sm"></q-btn>
        </div>
    </q-item-section>
</q-card>
```

Take a look at the picture to make it clearer what I mean:
https://1drv.ms/u/s!AhMmuIyzqnNsjpFGrTc4es7Xo87Efw?e=BvnOmY

**Comment by ontwikkelfabriek** [comment]
@SaleCar 
If you create a codepen.io we can provide better support for your problem:
https://codepen.io/rstoenescu/pen/VgQbdx

**Reply by sasgosh** [reply]
Hello,

Here is codepen. But, breadcrump is displayed differently. Slash is missing, I don't know why. 
https://codepen.io/SaleCar/pen/LYWWvgZ

**Reply by metalsadman** [reply]
close your tags in umd `</q-breadcrumbs-el>`.

**Comment by rstoenescu** [comment, marked as answer]
There you go: https://codepen.io/rstoenescu/pen/dyvROMO
Few things:

* Why using QItemSection? Doesn't makes sense
* Familiarize yourself with the flex CSS helper classes (it makes a world of difference) --- docs > Layout and Grid > Flex Grid

**Reply by sasgosh** [reply]
Thanx
